### PR TITLE
WINTERMUTE: Add detection entry for german Sunrise version

### PR DESCRIPTION
**Pull request by GunnarBirke**
Detection entry for the full version of the Wintermute game Sunrise in german.

### CODE REVIEW

File: engines/wintermute/detection_tables.h [hunk 1]
```diff
@@ -5137,6 +5137,11 @@ static const WMEGameDescription gameDescriptions[] = {
 	WME_WINENTRY("strangechange", "",
 		WME_ENTRY1s("data.dcp", "818e53c1584dac28f336195d7dcfb97d", 3762512), Common::EN_ANY, ADGF_UNSTABLE | GF_3D, WME_1_8_6),
 
+	// Sunrise: The game (German)
+	// NOTE: This is a 2.5D game that is out of ScummVM scope
```
Comment (aquadran) [inline engines/wintermute/detection_tables.h:5141]: this is not true any more :)

File: engines/wintermute/detection_tables.h [hunk 2]
```diff
@@ -5137,6 +5137,11 @@ static const WMEGameDescription gameDescriptions[] = {
...
+	WME_WINENTRY("sunrise", "",
+		WME_ENTRY1s("data.dcp", "d06744fc60982a30e82d96075698c8bc", 134409552), Common::DE_DEU, ADGF_UNSTABLE | ADGF_DEMO | GF_3D, WME_1_7_93),
```
Comment (aquadran) [inline engines/wintermute/detection_tables.h:5143]: remove demo flag

Comment (aquadran) [reply engines/wintermute/detection_tables.h:5143]: does it match exe version?

Comment (GunnarBirke) [reply engines/wintermute/detection_tables.h]: Demo flag removed. Wintermute version of the exe is 1.7.93, same as the demo.

File: engines/wintermute/detection_tables.h (hunk 1 above)
Comment (GunnarBirke) [reply engines/wintermute/detection_tables.h]: Right

**Pull request merged**